tcms/search/__init__.py

```diff
@@ -25,18 +25,16 @@
 def advance_search(request):
     """View of /advance-search/"""
     data = request.GET
-    target = data.get('target')
     plan_form = PlanForm(data)
     case_form = CaseForm(data)
     run_form = RunForm(data)
     # Update MultipleModelChoiceField on each form dynamically
     plan_form.populate(data)
     case_form.populate(data)
     run_form.populate(data)
-    all_forms = (plan_form, case_form, run_form)
```

Comment:
Do not delete this. We don't get any benefit from removing this variable instead the tuple is now used in 2 distinctive places: on line 37 and line 60. This will lead to bugs once somebody modifies one of the two lines and forgets about the other. 

Revert back to using this temporary variable.